app/cafe.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+
+        if expiration_date < datetime.now().date():
```

Comment:
Ensure that `expiration_date` is a `date` object before comparing it with `datetime.now().date()`. If `expiration_date` is a string, you should parse it into a `date` object.